object/search.go

```diff
@@ -299,3 +301,41 @@ func (f *SearchFilters) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+// AddPayloadHashFilter adds filter by payload hash.
+// `sum` type must be github.com/nspcc-dev/neofs-sdk-go/checksum.SHA256
```

Comment:
I think if you're to just reference `checksum.SHA256` (without full package name) it should work, https://go.dev/doc/comment.